app/src/main/java/org/schabi/newpipe/database/feed/dao/FeedDAO.kt

```diff
@@ -122,24 +47,37 @@ abstract class FeedDAO {
         INNER JOIN feed f
         ON s.uid = f.stream_id
 
-        INNER JOIN feed_group_subscription_join fgs
+        LEFT JOIN feed_group_subscription_join fgs
         ON fgs.subscription_id = f.subscription_id
 
-        WHERE fgs.group_id = :groupId
+        WHERE (
+            :groupId == ${FeedGroupEntity.GROUP_ALL_ID}
```

Comment:
Is there a reason why there are two ``=``?
Is this a typo?

Reply:
Uuups, I mixed java with sql (but in theory it should work)